app/src/main/java/com/nextcloud/talk/remotefilebrowser/activities/RemoteFileBrowserActivity.kt

```diff
@@ -139,11 +139,7 @@ class RemoteFileBrowserActivity : AppCompatActivity(), SelectionInterface, Swipe
                         onItemClicked(remoteFileBrowserItem)
                     }
                         .apply {
-                            items = if (remoteFileBrowserItems.size > 1) {
-                                remoteFileBrowserItems.subList(1, remoteFileBrowserItems.size)
-                            } else {
-                                ArrayList()
-                            }
+                            items = remoteFileBrowserItems
```

Comment:
this removes skipping the first item in case the list has more than one item, because the 1st item is the folder you are in and should thus be skipped. Not sure though it we shouldn't already remove it in the remote operation when return the result. What do you think @AlvaroBrey ?

Reply:
Sorry, I moved it in https://github.com/nextcloud/talk-android/pull/2086/commits/f3b5dc1df5ca8b2e769c9554693baf1cb9b5248a because I thought that  it was always root, not current folder. But yeah, it makes more sense to remove it in operation/repository. I would remove by path rather than the first element

Reply:
Pushed https://github.com/nextcloud/talk-android/pull/2086/commits/4acf71e14fe875eb3245744db5c673c2875eac92 to remove the extra filtering then.